### ISSUE TITLE
Search and intl php module in v1.9.4

### ISSUE THREAD

**Issue by jdnevado** [osTicket/osTicket#1454]
hi, i've intalled v1.9.4 in my server:
![osticketenv](https://cloud.githubusercontent.com/assets/9477963/4858311/ba6c8792-60dc-11e4-8cff-e95bf2aa5162.PNG)
Installation is ok and osticket is running well, but when i try to search text in subject or content all searchs are empty.
i notice that ost__search table rows content is like that:
![ost__search](https://cloud.githubusercontent.com/assets/9477963/4858409/b850f8ac-60dd-11e4-99bc-e29d63d25bd3.PNG)
Looking at the code i found that:

``` php
        if (class_exists('IntlBreakIterator')) {
            // Split by word boundaries
            if ($tokenizer = IntlBreakIterator::createWordInstance(
                    $lang ?: ($cfg ? $cfg->getSystemLanguage() : 'en_US'))
            ) {
                $tokenizer->setText($text);
                $tokens = array();
                foreach ($tokenizer as $token){
                    $tokens[] = $token;}
                $text = implode(' ', $tokens);
            }
        }
```

in foreach sentence, $token is a int who mark position of a break char. If i disable intl mod (class_exists('IntlBreakIterator') is false) the search run again and ost__search contents are strings.

how important is mod_intl for osticket?

thanks in advance.

**Comment by amsut** [comment]
@greezybacon just wondering whether this issue will be fixed anytime soon? Thank you :)

**Comment by greezybacon** [comment]
This was retired in the `develop` branch to be released in v1.9.6.1
